.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- Please fill out as much of this as you can. Thanks! -->
+
+### Description
+<!--- Describe your changes -->
+
+### Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+### How Has This Been Tested?
+<!--- Please describe how you tested your changes. -->
+<!--- It's okay to admit it hasn't been robustly tested. -->
+
+### Screenshots (if appropriate):
+
+### Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+### Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+- [ ] I have [signed the CLA](https://code.facebook.com/cla) right now or previously.
+- [ ] I have read the [**CONTRIBUTING** document](https://github.com/facebook/AsyncDisplayKit/blob/master/CONTRIBUTING.md.
+- [ ] I have titled my pull request in the style of other PRs, e.g. "[ASTextNode] Fix..."
+- [ ] My change requires updates to the AsyncDisplayKit.org documentation.
+- [ ] All new and existing tests passed.
```

Comment:
The tests passed line is more trouble than its worth. GitHub will make it really clear whether the tests have passed for each commit, and they may pass/fail/pass/fail between different commits.